websites/J/JavaScript/presence.ts

```diff
@@ -0,0 +1,45 @@
+const presence = new Presence({
+    clientId: "786389332042711040"
+}), Stamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+    const data: PresenceData = {
+        largeImageKey: "jslogo",
+        startTimestamp: Stamp
+    }, paths = document.location.pathname.split('/');
+    paths.splice(0, 1);
+
+    if(paths[3] !== 'JavaScript')return;
+
+    if(paths[0] === 'es'){
+        if(!paths[4]){
+            data.details = `Viendo la pagina principal...`;
+        } else {
+            data.details = `Viendo ${paths[4]}`;
+            paths.splice(0, 1);
+            paths.splice(0, 1);
+            paths.splice(0, 1);
+            paths.splice(0, 1);
+            paths.splice(0, 1);
+            if(paths[0]){
+                data.state = `Tema: ${paths.join(', ')}`;
+            }
+        }
+    } else {
+        if(!paths[4]){
+            data.details = `Looking the main page...`;
+        } else {
+            data.details = `Looking at ${paths[4]}`;
+            paths.splice(0, 1);
+            paths.splice(0, 1);
+            paths.splice(0, 1);
+            paths.splice(0, 1);
+            paths.splice(0, 1);
+            if(paths[0]){
+                data.state = `Topic: ${paths.join(', ')}`;
+            }
+        }
+    }
```

Comment:
and what you did there? How it is gonna work tho? Screenshots? Also, don't add only support for ``es`` and english one. Just wait for our extension update that will have multilang support. For now do support only for english

Reply:
okay

Reply:
First the code remove a empty element of the array, then because the url is developer.mozilla.org, the code checks if its the docs of JavaScript, then the code checks if the page its in spanish or in english, then if there is no paths[4] that its "Reference, Guide" or something like that the code says that it sin the main page, then if it is more paths the code put that the category of the thing that the person is seeing and then the code removes this "/es/docs/Web/JavaScript/" that its the reason because i put 5 "paths.splice(0, 1);", then i use join to put the topic that the person is seeing

Reply:
Still it isn't good to do this. Change it to different way and remove es support.

Reply:
> Still it isn't good to do this. Change it to different way and remove es support.

i remove the es support, and i dont know a different way let me think